<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Manasa" failures="1" tests="2" name="testRunners.TestRunner" time="7.896" errors="0" timestamp="2025-07-24T14:44:11 CEST" skipped="0">
  <testcase classname="testRunners.TestRunner" name="runScenario" time="2.969">
    <failure message="Navigated to checkout without adding any items expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Navigated to checkout without adding any items expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepDefinitions.LoginSteps.userDoNotAddProductToCartAndPlaceOrder(LoginSteps.java:87)
at âœ½.User do not add product to cart and place order(file:///E:/Manasa/Interview/Selenium_Java_Automation/Exact_Automation_Assesment/src/test/resources/features/Testing.feature:21)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunners.TestRunner" name="runScenario" time="4.927"/>
</testsuite> <!-- testRunners.TestRunner -->
